dlnd/cime_config/namelist_definition_dlnd.xml

```diff
@@ -56,11 +56,10 @@
     <input_pathname>abs</input_pathname>
     <group>dlnd_nml</group>
     <desc>
-      file specifying model mask if not obtained from input model mesh
+      file specifying file to use to obtain model mask
     </desc>
     <values>
-      <value>$LND_DOMAIN_MESH</value>
-      <value set_model_maskfile='true'>$LND_DOMAIN_PATH/$LND_DOMAIN_FILE</value>
+      <value>$MASK_MESH</value>
```

Comment:
I just wonder that does this require some changes in the nems.configure in the HAFS side? Do we need to add this attribute to the file or it is just optional. Its I see dlnd and docn involves and i could not be sure about its side effect in HAFS app.

Reply:
@uturuncoglu  - that's a really good questions.
Right now it only changes the docn and dlnd configurations - so it should not affect what you are trying to do with HAFS. Maybe you can confirm that?